### PR TITLE
reduce unittest gpu memory

### LINKED ISSUE #3437: Control GPU memory usage in unit tests
@emailweixu found that tests can fail for memory allocation if running them in parallel `ctest -I 121,123 -j`:

```
I found that tests can fail for memory allocation if running them in parallel: ctest -I 121,123 -j
121: Traceback (most recent call last):
121:   File "/home/wei/code/baidu/idl/dl/robot/external/paddle/python/paddle/v2/framework/tests/op_test_util.py", line 37, in test_all
121:     var.set(arr, place)
121: RuntimeError: Insufficient GPU memory to allocation. at [/home/wei/code/baidu/idl/dl/robot/external/paddle/paddle/framework/tensor.h:131]
121: Call Stacks:
121: /home/wei/code/baidu/idl/dl/robot/build/.env/local/lib/python2.7/site-packages/paddle/v2/framework/core.so(_ZN6paddle8platform13EnforceNotMetC1ENSt15__exception_ptr13exception_ptrEPKci+0x1d2) [0x7fde36115fc2]
121: /home/wei/code/baidu/idl/dl/robot/build/.env/local/lib/python2.7/site-packages/paddle/v2/framework/core.so(_ZN6paddle9framework6Tensor15PlaceholderImplIfNS_8platform8GPUPlaceEEC1ES4_m+0x130) [0x7fde36136824]
121: /home/wei/code/baidu/idl/dl/robot/build/.env/local/lib/python2.7/site-packages/paddle/v2/framework/core.so(_ZN6paddle9framework6Tensor12mutable_dataIfEEPT_N5boost7variantINS_8platform8GPUPlaceEJNS7_8CPUPlaceEEEE+0x1b1) [0x7fde3612bbf9]
....
1/3 Test #123: test_softmax_op ..................   Passed    1.31 sec
2/3 Test #122: test_sigmoid_op ..................***Failed    1.40 sec
3/3 Test #121: test_add_two_op ..................***Failed    1.42 sec
```

### PR DESCRIPTION
**Pull request by QiJune**
fix #3437

### CODE REVIEW

File: python/paddle/v2/framework/tests/test_tensor.py [hunk 1]
```diff
@@ -11,10 +11,10 @@ def test_int_tensor(self):
 
         tensor = var.get_tensor()
 
-        tensor.set_dims([1000, 784])
+        tensor.set_dims([100, 84])
```
Comment (wangkuiyi) [inline python/paddle/v2/framework/tests/test_tensor.py]: Can you try three configurations -- [7, 0], [0, 7], [7,13]  -- 7 and 13 are just small prime numbers.  Prime numbers are reasonable becaue they are not power of 2 or any way defactorizable. 0 is a good boundary case.

File: python/paddle/v2/framework/tests/test_softmax_op.py [hunk 2]
```diff
@@ -18,7 +18,7 @@ class TestSoftmaxOp(unittest.TestCase):
 
     def setUp(self):
         self.type = "softmax"
-        self.inputs = {'X': np.random.random((32, 100)).astype("float32")}
+        self.inputs = {'X': np.random.random((32, 22)).astype("float32")}
```
Comment (wangkuiyi) [inline python/paddle/v2/framework/tests/test_softmax_op.py]: Let us use small prime numbers.

File: python/paddle/v2/framework/tests/test_cross_entropy_op.py [hunk 3]
```diff
@@ -24,7 +24,7 @@ def setUp(self):
 class CrossEntropyGradOpTest(GradientChecker):
     def test_softmax_grad(self):
         op = create_op("onehot_cross_entropy")
-        batch_size = 100
+        batch_size = 32
```
Comment (wangkuiyi) [inline python/paddle/v2/framework/tests/test_cross_entropy_op.py]: Please add test case with batch size = 0./

File: python/paddle/v2/framework/tests/test_add_two_op.py [hunk 4]
```diff
@@ -13,8 +13,8 @@ class TestAddOp(unittest.TestCase):
     def setUp(self):
         self.type = "add_two"
         self.inputs = {
-            'X': numpy.random.random((102, 105)).astype("float32"),
-            'Y': numpy.random.random((102, 105)).astype("float32")
+            'X': numpy.random.random((12, 15)).astype("float32"),
```
Comment (wangkuiyi) [inline python/paddle/v2/framework/tests/test_add_two_op.py]: Please add cases like 

- [7, 0]
- [0, 7]
- [0, 0]
- [7, 13]  

7  and 13 are just small prime numbers.

**Comment by luotao1** [comment]
感谢您给PaddlePaddle贡献代码。由于Paddle V1/V2版本已不再维护，相关代码也已从develop分支上删除，因此关闭您的PR，欢迎您向Paddle最新版-Fluid贡献代码。
Thanks for contributing to PaddlePaddle! Since V1/V2 will not be maintained anymore, and related codes have been deleted from develop branch as well, we close this PR. Welcome to contribute to Fluid——the latest version of PaddlePaddle.
